src/formio/components/TextField.js

```diff
@@ -1,10 +1,14 @@
 import { Formio } from 'react-formio';
 
 import { applyPrefix } from '../utils';
-
+import {get} from "../../api";
```

Comment:
Please keep the code style consistent with the existing code: single quotes for strings, space between curly braces. I'll be setting up `prettier` to enforce this in the near future.

```suggestion
import { get } from '../../api';
```